maps/EusinesHouse.asm

```diff
@@ -183,3 +187,14 @@ EusinesHouseGrampsText2:
 	line "about the legend-"
 	cont "ary #mon!"
 	done
+
+EusinesHouseGrampsText3:
+	text "My grandson Eusine"
+	line "is on another"
+	cont "adventure."
+
+	para "He left to visit"
```

Comment:
This seems to imply you'd see him in Ecruteak, but you shouldn't.

Reply:
Should it be more specific and say that he left to visit the Tin Tower in Ecrutreak City?

Reply:
If he's in Bell Tower at that time, yes.